biodatasets/genia_term_corpus/genia_term_corpus.py

```diff
@@ -0,0 +1,377 @@
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+The identification of linguistic expressions referring to entities of interest in molecular biology such as proteins,
+genes and cells is a fundamental task in biomolecular text mining. The GENIA technical term annotation covers the
+identification of  physical biological entities as well as other important terms. The corpus annotation covers the full
+1,999 abstracts of the primary GENIA corpus.
+"""
+
+import xml.etree.ElementTree as ET
+from itertools import count
+from typing import Dict, List, Tuple
+
+import datasets
+
+from utils import schemas
+from utils.configs import BigBioConfig
+from utils.constants import Tasks
+
+_CITATION = """\
+@inproceedings{10.5555/1289189.1289260,
+author = {Ohta, Tomoko and Tateisi, Yuka and Kim, Jin-Dong},
+title = {The GENIA Corpus: An Annotated Research Abstract Corpus in Molecular Biology Domain},
+year = {2002},
+publisher = {Morgan Kaufmann Publishers Inc.},
+address = {San Francisco, CA, USA},
+booktitle = {Proceedings of the Second International Conference on Human Language Technology Research},
+pages = {82–86},
+numpages = {5},
+location = {San Diego, California},
+series = {HLT '02}
+}
+
+@article{Kim2003GENIAC,
+  title={GENIA corpus - a semantically annotated corpus for bio-textmining},
+  author={Jin-Dong Kim and Tomoko Ohta and Yuka Tateisi and Junichi Tsujii},
+  journal={Bioinformatics},
+  year={2003},
+  volume={19 Suppl 1},
+  pages={
+          i180-2
+        }
+}
+
+@inproceedings{10.5555/1567594.1567610,
+author = {Kim, Jin-Dong and Ohta, Tomoko and Tsuruoka, Yoshimasa and Tateisi, Yuka and Collier, Nigel},
+title = {Introduction to the Bio-Entity Recognition Task at JNLPBA},
+year = {2004},
+publisher = {Association for Computational Linguistics},
+address = {USA},
+booktitle = {Proceedings of the International Joint Workshop on Natural Language Processing in Biomedicine and Its
+Applications},
+pages = {70–75},
+numpages = {6},
+location = {Geneva, Switzerland},
+series = {JNLPBA '04}
+}
+"""
+
+_DATASETNAME = "genia_term_corpus"
+
+_DESCRIPTION = """\
+The identification of linguistic expressions referring to entities of interest in molecular biology such as proteins,
+genes and cells is a fundamental task in biomolecular text mining. The GENIA technical term annotation covers the
+identification of  physical biological entities as well as other important terms. The corpus annotation covers the full
+1,999 abstracts of the primary GENIA corpus.
+"""
+
+_HOMEPAGE = "http://www.geniaproject.org/genia-corpus/term-corpus"
+
+_LICENSE = """GENIA Project License for Annotated Corpora
```

Comment:
Let's just use "GENIA Project License for Annotated Corpora" here.

Reply:
Done @jason-fries.